cmd/config/ext/ext.go

```diff
@@ -3,22 +3,8 @@
 
 package ext
```

Comment:
I feel this should be a part of more basic package like kyaml so that it is accessible from everywhere and cmd/config need not pass it to dependencies. WDYT?

Reply:
I considered this, but I don't think we want to push this into `kyaml` now. Passing it to dependencies should be easier when we can get the name without having to worry about errors.